test/Twig/Tests/Extension/HumanReadableBytesTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+require_once __DIR__.'../../../../lib/Twig/Extensions/Extension/HumanReadableBytes.php';
+
+class HumanReadableBytesTest extends \PHPUnit_Framework_TestCase
+{
+    protected $humanReadableBytesExtension;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->humanReadableBytesExtension = new HumanReadableBytes();
+    }
+
+    public function getHumanReadableBytesTestData()
+    {
+        return [
```

Comment:
the short array syntax is not supported in older PHP versions which are still supported by this library